guides/common/modules/proc_increasing-host-limit-for-pull-based-rex-transport.adoc

```diff
@@ -0,0 +1,34 @@
+[id="Increasing_host_limit_for_pull-based_REX_transport_{context}"]
+= Increasing Host Limit for Pull-Based REX Transport
+
+{Project} has a pull-based transport that has the features of REX but with a deployment model similar to katello-agent. 
+This transport uses MQTT as its messaging protocol and includes an MQTT client running on each host.
+To tune the MQTT server `mosquitto` and increase the number of hosts connected to it, follow this procedure on your {ProjectServer}:
+
+.Procedure
+. Enable pull-based remote execution on your {ProjectServer} or {SmartProxyServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode pull-mqtt
+----
++
+Note that your {ProjectServer} or {SmartProxyServer} can only use one transport mode, either SSH or MQTT.
```

Comment:
alternative: instead of having users to enable Pull mode, maybe make it a prerequisite? I doubt that anyone follows this procedure without actually already using REX Pull mode yet.

Reply:
We can do that as well but It is a good idea to explain about the entire procedure  (as we are doing here) so that customer or user is aware of it :)